docs/dev/plugins/cloudformation/cloudformation-basics.md

```diff
@@ -0,0 +1,37 @@
+# AWS CloudFormation
+How to transform a TOSCA model to an Amazon Web Services CloudFormation template.
+
+## Mapping
+see [this](mapping.md)
```

Comment:
Maybe elaborate a bit more on what is being linked here.
Just "see this" seems kind of weird.